rust-runtime/aws-smithy-runtime-api/src/client/identity/http.rs

```diff
@@ -52,6 +52,12 @@ impl Token {
     pub fn expiration(&self) -> Option<SystemTime> {
         self.0.expiration
     }
+
+    /// Creates a `Token` for tests.
+    #[cfg(feature = "test-util")]
+    pub fn for_tests() -> Self {
+        Self::new("test-token", None)
+    }
```

Comment:
`aws-config::ConfigLoader::with_test_credentials` should set a token identity as well

Reply:
Do you mean the `test_credentials` method? I do have the generated `Config::with_test_defaults()` method setting a token.

Reply:
https://docs.rs/aws-config/latest/aws_config/struct.ConfigLoader.html#method.test_credentials